packages/opencensus-core/src/stats/view.ts

```diff
@@ -181,15 +193,55 @@ export class BaseView implements View {
    * @param bucketBoundaries a list with the bucket boundaries
    */
   private createBuckets(bucketBoundaries: number[]): Bucket[] {
-    return bucketBoundaries.map((boundary, boundaryIndex) => {
-      return {
-        count: 0,
-        lowBoundary: boundaryIndex ? boundary : -Infinity,
-        highBoundary: (boundaryIndex === bucketBoundaries.length - 1) ?
-            Infinity :
-            bucketBoundaries[boundaryIndex + 1]
-      };
-    });
+    const boundaries = bucketBoundaries.filter(Math.clz32);
```

Comment:
This doesn't seem to catch all negative numbers. For example, this passes through `-Infinity`. Is that ok?

Reply:
It also seems a little compute wasteful to do this `.filter` just for a console log. Can't the logging done during the reduce?

Reply:
Improved. Thank you